docker/dev_env/entrypoint.sh

```diff
@@ -0,0 +1,76 @@
+#! /usr/bin/env bash
+
+set -e
+
+if [ "$UID" = "0" ]; then
+  printf "Do not run this container with the root user! Use the 'run.sh' script or pass --user to docker run.\n"
+  exit 1
+fi
```

Comment:
This bit had to be dropped because the Mac-compatible implementation cannot supply a reasonable `--user` argument to Docker.